### PR TITLE
Servicability issues

### PR DESCRIPTION
**Pull request by mbwhite**
This moves the filtering into the common code, didn't mean to be as big a change as this. 
But by making the filtering work as intended in the common code, it means that the runtimes
can be simpler. Today they currently just use console.log

We have a console.log formatter already. 
  

Resolves issue #2993

### CODE REVIEW

File: packages/composer-cli/lib/cmds/network/loglevelCommand.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const logLevel = require ('./lib/loglevel.js');
 module.exports.command = 'loglevel [options]';
 module.exports.describe = 'Change the logging level of a business network';
 module.exports.builder = {
-    newlevel: { alias: 'l', optional: true, describe: 'the new logging level', choices : ['INFO', 'WARNING', 'ERROR', 'DEBUG'] },
+    newlevel: { alias: 'l', optional: true, describe: 'the new logging level', type: 'string'},//choices : ['INFO', 'WARNING', 'ERROR', 'DEBUG'] },
```
Comment (davidkel) [inline packages/composer-cli/lib/cmds/network/loglevelCommand.js]: commented out code

File: packages/composer-playground/src/assets/npmlist.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (davidkel) [inline packages/composer-playground/src/assets/npmlist.json]: Don't think this file should be part of the pull request

File: packages/composer-runtime-hlfv1/lib/composer.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ class Composer {
         let engine = this._createEngine();
         let nodeContext = this._createContext(engine, stub);
         try {
-            await this.container.initLogging(stub);
+            // await this.container.initLogging(stub);
```
Comment (davidkel) [inline packages/composer-runtime-hlfv1/lib/composer.js]: You have to do this, we need access to the stub

File: packages/composer-runtime-hlfv1/lib/composer.js [hunk 4]
```diff
@@ -90,7 +90,7 @@ class Composer {
...
```
Comment (davidkel) [inline packages/composer-runtime-hlfv1/lib/composer.js]: You need to do this, you need access to the stub.

File: packages/composer-runtime-hlfv1/lib/nodeloggingservice.js [hunk 5]
```diff
@@ -16,168 +16,30 @@
...
-    500: 'DEBUG'
-};
-
-const LOGLEVEL_KEY = 'ComposerLogLevel';
-
 /**
  * Base class representing the logging service provided by a {@link Container}.
  * @protected
  */
 class NodeLoggingService extends LoggingService {
```
Comment (davidkel) [inline packages/composer-runtime-hlfv1/lib/nodeloggingservice.js]: Stripping out this file will stop the logging of the transaction id. I'm not sure that there would ever be a need to do anything except console logging, winston maybe overkill, plus we may want to consider combining with the fabric shim logging

Comment (davidkel) [inline packages/composer-runtime-hlfv1/lib/nodeloggingservice.js]: Also, we need to be able to get the CHAINCODE environment variable still, that proved to be indispensible over the xmas period. Also we need to ensure that any change of log level on the chaincode is stored in the world state.

**Comment by sstone1** [comment]
No action for nearly a month; closing.

**Pull request closed without merging**